google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java

```diff
@@ -1440,4 +1443,19 @@ public void testReadCompressedBlob() throws IOException {
     }
     blob.delete();
   }
+
+  @Test
+  public void testBlobNameNormalization() {
+    for (String blobName : BLOB_NORMALIZATION.keySet()) {
+      BlobInfo blobInfoFormC = BlobInfo.newBuilder(BlobId.of(BUCKET, blobName))
```

Comment:
Oops, I started with using hardcoded variables and switched to using this loop, then I forgot to change the name back....

Done.